fix: OPTIC-2155: Info tab is not optimized for dark mode for TimeSeriesLabel with underlying Choices

ricardoantoniocm opened the pull request:
### Reason for change

The text displayed in the Info tab for TimeSeriesLabel selections was not optimized for dark mode, resulting in black text on a dark background, which reduces readability. This change addresses this issue by ensuring the text color adapts appropriately for both light and dark modes, improving the user experience.

### Screenshots

**Before:**
<img width="287" alt="image" src="https://github.com/user-attachments/assets/062558b8-c7b4-4cf7-ab40-66c7f6dd692b" />
<img width="307" alt="image" src="https://github.com/user-attachments/assets/6710303e-fdc9-4daa-ae71-1c5c021525b8" />


**After:**
<img width="269" alt="image" src="https://github.com/user-attachments/assets/3c564011-2303-45ef-a431-f7bc677978df" />
<img width="304" alt="image" src="https://github.com/user-attachments/assets/5a0dadd9-1360-4ff4-adf0-9b5444d3a47d" />


### Rollout strategy

This is a small visual change and does not require a specific rollout strategy. The changes will be deployed with the next release.

### Testing

1.  **Verify Dark Mode:**
    * Navigate to the LSO/LSE application in dark mode.
    * Create a project with the "Signal Quality" template.
    * Create a label, select it and choose an option from the Choices.
    * Open the Info tab.
    * Confirm that the text for both the key and value pairs is easily readable against the dark background (using the `color-neutral-content` color).
    * Confirm that the background of the value is a lighter color (`color-primary-background`) providing sufficient contrast.
    * Verify the added padding and rounded corners improve the visual consistency.

2.  **Verify Light Mode:**
    * Switch to Light Mode.
    * Confirm that the text and background colors in the Info tab are appropriate and readable in light mode.

### Risks

There are minimal risks associated with this change as it primarily involves styling adjustments. However, potential risks include:

* **Unexpected visual regressions:** The new styles might inadvertently affect other elements in the application. Thorough testing in both light and dark modes should mitigate this.

### Reviewer notes

* Please pay close attention to the contrast ratio between the text and background in both light and dark modes to ensure accessibility.
* Verify that the added padding and rounded corners are applied consistently with other similar elements in the application.
* Confirm that no other unintended visual changes have been introduced.

### General notes

* The text color for the key and value pairs has been set to the `color-neutral-content` token.
* The background color for the value has been set to the `color-primary-background` token.
* Padding and rounded corners have been added to the value for better visual consistency.

ricardoantoniocm committed:
fix: OPTIC-2155: Info tab is NOT optimized for Dark mode when TimeSeriesLabel is selected

ricardoantoniocm committed:
Fixes Info panel elements not adapted to Dark Mode when a label with a nested Choices element is selected.

ricardoantoniocm commented on web/libs/editor/src/components/SidePanels/DetailsPanel/RegionDetails.scss:
```diff
@@ -132,6 +132,18 @@
+    color: var(--color-neutral-content);
+
+    & .ant-typography {
+      color: var(--color-neutral-content);
```
Cannot be consolidated unfortunately.

AndrejOros commented:
/fm sync 
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14902091759)

AndrejOros committed:
Merge branch 'develop' into 'fb-optic-2155/info-tab-is-not-optimized-for-dark-mode'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14902091759

AndrejOros committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14902091759

ricardoantoniocm commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14932344278)
> Successfully merged:  rename web/libs/datamanager/src/components/App/{App.jsx => App.tsx} (73%)

ricardoantoniocm committed:
Merge branch 'develop' into 'fb-optic-2155/info-tab-is-not-optimized-for-dark-mode'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14932344278
